app/(service)/my-store/[storeId]/_components/myStoreCard.tsx

```diff
@@ -0,0 +1,67 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import { useParams } from 'next/navigation';
+
+import Button from '@/components/button/button';
+import { getMyStoreData } from '@/services/api';
+
+/**
+ * @todo 처음에 가게 정보 없을 때 보여줄 가게없음 보이게 => 없을 때에는 가게 등록하기 보이게
+ * @todo 편집하기 누르면 가게 정보 편집 페이지로
+ * @todo 공고 등록하기 페이지를 누르면 공고 등록하기 페이지로 이동
+ * @todo 내가 등록한 공고 무한스크롤로, 최신순으로 정렬하고,
+ */
+export default function MyStoreCard() {
+  const { storeId } = useParams();
+  const [storeData, setStoreData] = useState({ imageUrl: '', category: '', name: '', address1: '', description: '' });
+
+  useEffect(() => {
+    (async () => {
+      const { item } = await getMyStoreData(storeId as string);
+      setStoreData(item); // 이렇게 하면 item을 컴포넌트의 상태로 저장하고, 컴포넌트의 전체 범위에서 사용할 수 있습니다.
```

Comment:
P3: 이 부분은 주석을 달지 않아도 괜찮을 것 같습니다!

```suggestion
      setStoreData(item);
```